tiktoken-rs/src/api.rs

```diff
@@ -163,6 +165,7 @@ pub fn num_tokens_from_messages(
 ///
 /// If successful, the function returns a `Result` containing the maximum number of tokens available for chat completion,
 /// based on the given model and messages.
+#[cfg(feature = "async-openai")]
 pub fn get_chat_completion_max_tokens(
```

Comment:
I've pushed a new commit, but supporting two versions of the same functionality feels a bit awkward:

- I've named the async-openai version with an `async_openai` prefix. Please share your thoughts on this naming, or suggest a better alternative.
- The locally defined `ChatCompletionRequestMessage` differs from the one in the async-openai crate, requiring a clone of messages for each call into the base method.

I'm inclined to keep just one version, but that would break compatibility.